packages/manager/modules/pci/src/projects/project/storages/databases/components/tags-input/tags-input.html

```diff
@@ -1,21 +1,25 @@
-<oui-field data-label="{{ $ctrl.label }}" data-size="xl">
-    <div class="oui-input-group oui-input-group_xl">
-        <input
-            class="oui-input"
-            type="text"
-            id="tagInput"
-            name="tagInput"
-            ng-model="$ctrl.tag"
-            maxlength="255"
-            ng-keypress="$ctrl.handleKeyPressed($event)"
-        />
-        <oui-button disabled="$ctrl.isDisabled()" ng-click="$ctrl.addTag()"
-            ><span data-ng-bind="$ctrl.buttonLabel"></span
-        ></oui-button>
-    </div>
-    <oui-chips
-        items="$ctrl.items"
-        on-remove="$ctrl.removeTag(removed)"
-        closable
-    ></oui-chips>
-</oui-field>
+<form novalidate>
```

Comment:
if you add a form then t would make sense to also use an ng-submit

Reply:
Fixed here: https://github.com/ovh/manager/pull/5627/commits/d4c735fd418d97af581a0b760c05fd8825501442